pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tuspy"
+description = "A Python client for the tus resumable upload protocol ->  http://tus.io"
+authors = [
```

Comment:
If there is someone who is maintain the project that isn't the original author, they should consider adding themselves to the [maintainers](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#authors-maintainers) metadata.